exercises/concept/vehicle-purchase/.meta/exemplar.go

```diff
@@ -19,6 +19,7 @@ func ChooseVehicle(first, second string) string {
 // CalculateResellPrice calculates how much a vehicle can resell for at a certain age.
 func CalculateResellPrice(originalPrice, age float64) float64 {
 	var percentage float64
+	//nolint:gocritic // To disable ifElseChain. The point of this exercise is to teach if-else-if-else, not switch.
```

Comment:
The only way to work around the `ifElseChain` warning was to disable the `gocritic` checks. I wasn't sure if there was a way to disable just `ifElseChain` specifically? 

I tried `//nolint:gocritic,ifElseChain`, but I kept getting the warning `WARN [runner/nolint] Found unknown linters in //nolint directives: ifelsechain`

Reply:
Not that I know of.

Reply:
```suggestion
	// Disables ifElseChain rule of gocritic, which suggests convert the following code to a switch,
	// however the point of this exercise is to teach if-else-if-else, not switch.
	//nolint:gocritic
```